gunicorn/workers/base.py

```diff
@@ -132,7 +130,9 @@ def changed(fname):
             reloader_cls = reloader_engines[self.cfg.reload_engine]
             self.reloader = reloader_cls(extra_files=self.cfg.reload_extra_files,
                                          callback=changed)
-            self.reloader.start()
+
+        self.load_wsgi()
+        self.reloader.start()
```

Comment:
This needs to be wrapped in a conditional in case the reloader doesn't exist, right?

Reply:
You're right, I've fixed it.